swig/yui.i

```diff
@@ -269,8 +269,10 @@ class Exception;
   { return ($self != i); }
 #endif
 #if defined(SWIGPYTHON)
-  int __cmp__( YItem *i )
-  { return ($self - i); }
+  int __eq__( YItem *i )
+  { return ($self == i); }
+  int __ne__( YItem *i )
+  { return ($self != i); }
```

Comment:
well, difference between cmp and eq/ne is that cmp can also do operations for `>` `>=` `<` `<=` is it problem if it no longer works? From my POV not so big, but I do not use python.